packages/webapp/src/style/index.js

```diff
@@ -0,0 +1,6 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+import '@jupyterlab/notebook/style/index.js';
+import './base.css';
```

Comment:
Does this reorg have a purpose in this package (private)?

Reply:
It will be removed.